packages/client/src/tools.js

```diff
@@ -882,8 +883,20 @@ export default class Tools {
   static extractAbstract(url) {
     const metaExtractionUrl = 'https://meta-extraction-wivlbyt6ga-uc.a.run.app/metadata';
     const queryString = generateQueryString({ url });
-    return fetch(`${metaExtractionUrl}?${queryString}`, {
+    const request = fetch(`${metaExtractionUrl}?${queryString}`, {
```

Comment:
You could also change the extractAbstract function to be async and use await here 😬, but it's not necessary.
```suggestion
    const result = await fetch(`${metaExtractionUrl}?${queryString}`, {
```